src/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant.Tests/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant.Tests.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\GSoft.AspNetCore.Authentication.ClientCredentialsGrant\GSoft.AspNetCore.Authentication.ClientCredentialsGrant.csproj" />
```

Comment:
The package `Microsoft.AspNetCore.Authentication.JwtBearer` on line 19 can be removed now that we reference our server-side project